### DISCUSSION TITLE
Serving dashboard  to the saleor core as bundle.

### DISCUSSION THREAD

**Discussion by erne15** [saleor/saleor#11628] [Deployment]
I want to  have dashboard  build served as bundle of static files but i have no idea how to edit the vite config.ts file as well as the settings.py in saleor backend. kindly is there anyone who as tried these.

**Comment by peelar** [comment]
Hello @erne15 👋 

`saleor-dashboard` is a static application. AFAIK, there is nothing you need to change in the config to build it as such. You can read more about it in the [`deployment.md`](https://github.com/saleor/saleor-dashboard/blob/main/docs/deployment.md) article.
